app/Controllers/feedController.php

```diff
@@ -416,14 +414,22 @@ public static function actualizeFeed(int $feed_id, string $feed_url, bool $force
 						throw new FreshRSS_Feed_Exception('XML+XPath parsing failed for [' . $feed->url(false) . ']');
 					}
 				} else {
-					$simplePie = $feed->load(false, $isNewFeed);
-					if ($simplePie === null) {
-						// Feed is cached and unchanged
-						$feedIsUnchanged = true;
-					}
+					$simplePie = $feed->load(false, $feedIsNew);
+				}
+
+				if ($simplePie === null) {
+					// Feed is cached and unchanged
+					$newGuids = [];
+					$entries = [];
+					$feedIsEmpty = false;	// We do not know
+					$feedIsUnchanged = true;
+				} else {
+					$newGuids = $feed->loadGuids($simplePie);
+					$entries = $feed->loadEntries($simplePie);
+					$feedIsEmpty = $simplePiePush !== null && empty($newGuids);
```

Comment:
Wrong boolean test. Regression fixed in https://github.com/FreshRSS/FreshRSS/pull/6663